WeightProducer/python/MCSamples_Summer20UL16.py

```diff
@@ -0,0 +1,105 @@
+import FWCore.ParameterSet.Config as cms
+
+from TreeMaker.WeightProducer.MCSample import MCSample
+
+# 13 TeV miniAOD samples - Summer20UL16
+Summer20UL16samples = [
+    # NB: amcatnlo samples have negative weight events, so NumberEvts = # positive - # negative
+    MCSample('WGJets_MonoPhoton_PtG-40to130_TuneCP5_13TeV-madgraph-pythia8', '106X_mcRun2_asymptotic_v17-v2', 'RunIISummer20UL16MiniAODv2', 'Constant', 1386165, False),
+    MCSample('WWG_TuneCP5_13TeV-amcatnlo-pythia8', '106X_mcRun2_asymptotic_v17-v1', 'RunIISummer20UL16MiniAODv2', 'Constant', 698000, False),
```

Comment:
amcatnlo samples should have the `NumberEvtsDiff` field filled because they have negative weights

Reply:
Good catch, I'll fix that one and see if I missed any others.

Reply:
Corrected in ea9d776